app/src/main/java/chat/rocket/android/chatroom/adapter/MessageViewHolder.kt

```diff
@@ -8,9 +9,9 @@ import kotlinx.android.synthetic.main.avatar.view.*
 import kotlinx.android.synthetic.main.item_message.view.*
 
 class MessageViewHolder(
-    itemView: View,
-    listener: ActionsListener,
-    reactionListener: EmojiReactionListener? = null
+        itemView: View,
```

Comment:
As per [Kotlin Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting), the old indentation was correct.

Reply:
They said it's 4 spaces. I'll change my tab spacing on AS to 4.